cpg_workflows/stages/large_cohort.py

```diff
@@ -109,6 +109,7 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
             gvcf_paths=new_sg_gvcfs,
             vds_paths=vds_paths,
             save_path=output_paths['combiner_plan'],
+            force_new_combiner=config_retrieve(['combiner', 'force_new_combiner'], False),
```

Comment:
As above, either have the default here or in the large_cohort.toml, but probably not both?

Reply:
Changed so now the user must set it via the config